<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="cell_width_label">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Ширина ячейки сетки</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="cell_width_spin_box">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>500000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cell_height_label">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Высота ячейки сетки</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="cell_height_spin_box">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>500000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="scale_label">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Масштаб (у.е.:метр)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="scale_spin_box">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="map_layout">
     <item>
      <widget class="QLabel" name="map_label">
       <property name="text">
        <string>Изображение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>549</width>
      <height>202</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="map_actions">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="load_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Загрузить
 изображение</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="open_map_visualization_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Открыть 3D
 визуализацию
 текущего объекта</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="save_map_as_image_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить карту
(.jpeg, .png)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="enter_heights_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>600</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Ввод высот
(при помощи 
программы)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="open_map_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Открыть карту в
новом окне</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="save_map_as_excel_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить карту
(.xlsx)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>251</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="update_grid_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Перестроить
сетку</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="update_map_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Перерисовать
область на
изображении</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="water_object_type_combo_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>220</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ratio_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Коэффициент А&lt;span style=&quot; vertical-align:sub;&quot;&gt;z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ratio_double_spin_box">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="enter_speed_vectors_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Ввести значения
векторов
 скоростей</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="computate_pressures_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Ввести значения
векторов
 скоростей</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>save_map_as_excel_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveMapAsExcelButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_map_as_image_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveMapAsImageButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_map_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>openMapButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>update_grid_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGridButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_map_visualization_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>openMapVisualizationButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadMapButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enter_heights_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterHeightsButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>update_map_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateFullMapButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enter_speed_vectors_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterSpeedVectorsButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadMapButtonPressed()</slot>
  <slot>openMapButtonPressed()</slot>
  <slot>saveMapAsExcelButtonPressed()</slot>
  <slot>saveMapAsImageButtonPressed()</slot>
  <slot>openMapVisualizationButtonPressed()</slot>
  <slot>enterHeightsButtonPressed()</slot>
  <slot>updateFullMapButtonPressed()</slot>
  <slot>updateGridButtonPressed()</slot>
  <slot>editGridButtonPressed()</slot>
  <slot>enterSpeedVectorsButtonPressed()</slot>
 </slots>
</ui>
